packages/features/embed/Embed.tsx

```diff
@@ -232,6 +241,15 @@ const EmailEmbed = ({
   if (!eventType) {
     return null;
   }
+  if (!selectedDuration) {
+    setSelectedDuration(eventType.length);
+  }
```

Comment:
When selected duration is first instantiated in parent component, eventtype data is not yet fetched and can be undefined, set it to eventType length in this case.